source/pug/views/rules-and-regulations.pug

```diff
@@ -2,11 +2,11 @@ include ../partials/header.pug
 
 .nsf-page
```

Comment:
Dunno if this page should have this classname, now that I see it. I don't think I intended that.